validator/build.py

```diff
@@ -164,11 +164,11 @@ def GenValidatorProtoascii(out_dir):
   assert re.match(r'^[a-zA-Z_\-0-9]+$', out_dir), 'bad out_dir: %s' % out_dir
 
   protoascii_segments = [open('validator-main.protoascii').read()]
-  extensions = glob.glob('extensions/*/0.1/validator-*.protoascii')
+  extensions = glob.glob('extensions/*/(0.1|1.0)/validator-*.protoascii')
```

Comment:
I agree. but the function you're changing is to generate validator protoascii. Is that still needed? I think we should always use 1.0 version `validator-amp-sticky-ad.protoascii`, but test both 0.1 version `validator-amp-sticky-ad.html` and 1.0 version `validator-amp-sticky-ad.html` ? Please correct me.